Waste Piles - Dataset Package

wenlanzhang opened the pull request:
waste .md file and data

andy-clarke-uofg reviewed (changes requested):
Hi Wenlan - great to see such strong progress on this so far.

Here's a few comments that require attention before we can merge the PR.

1. Your image files will work well inside the markdown file but we require square images for the info section of the UI. As a suggestion - you could take the first satellite images and create square images from them. Please take out your own color scheme - these will work better as plain satellite screenshots.

2. Please take out all evidence of your color scheme from any images. We add color automatically.

3. Your interpretability .md file is looking good. Please remove duplicate files from the PR.

4. Links within the interpretability .md file - including image links - should be relative to this repository so that they show up properly on GitHub.

5. We need a model-metadata.json file. You can check the documentation for details about what's included here but the information is already on the excel sheet we've been using.

6. We need a model-licensing.md file. This will be for internal use only at this point but should include the details of the license that you've selected. As per your email suggestion I think that the CC Non Commercial Attribution license is a good idea. You can paste the license information into this .md file alongside information about how to attribute the dataset to your work. 

7. Taking a look at the format of your model-outputs.csv file - it looks like the values you are providing are not in the correct format. In your dataset I'm seeing - `9876509.162535913,270606.1440290334` for `lat` - `lon` values. The correct formats are something like `36.939056099753,-1.1161267590278299`. Take a look at some other datasets in the repository for more information.

8. Please remove the ID column from your model-outputs.csv file

I think that's all for now - let me know if you'd like support with any of the above points.

wenlanzhang commented:
Hi Andy and Diego,

Thank you for your detailed feedback. I’ve gone through your suggestions and have addressed all the points except for #3 and #6:

#3 (Duplicate .md file): I’ve reviewed the files but I’m not sure which duplicate needs to be removed. Could you please point me to the specific file so I can take care of it?

#6 (model-licensing.md): I agree that the CC Non-Commercial Attribution license is suitable. However, I’m not entirely sure how to structure this file—would it be possible to share a template or an example from another repository for reference?

For the remaining points (1, 2, 4, 5, 7, 8), I’ve already implemented the changes as suggested. Please let me know if there’s anything else that needs adjustment. Thanks again for your suggestion!

Best regards,
Wenlan

> Hi Wenlan - great to see such strong progress on this so far.
> 
> Here's a few comments that require attention before we can merge the PR.
> 
> 1. Your image files will work well inside the markdown file but we require square images for the info section of the UI. As a suggestion - you could take the first satellite images and create square images from them. Please take out your own color scheme - these will work better as plain satellite screenshots.
> 2. Please take out all evidence of your color scheme from any images. We add color automatically.
> 3. Your interpretability .md file is looking good. Please remove duplicate files from the PR.
> 4. Links within the interpretability .md file - including image links - should be relative to this repository so that they show up properly on GitHub.
> 5. We need a model-metadata.json file. You can check the documentation for details about what's included here but the information is already on the excel sheet we've been using.
> 6. We need a model-licensing.md file. This will be for internal use only at this point but should include the details of the license that you've selected. As per your email suggestion I think that the CC Non Commercial Attribution license is a good idea. You can paste the license information into this .md file alongside information about how to attribute the dataset to your work.
> 7. Taking a look at the format of your model-outputs.csv file - it looks like the values you are providing are not in the correct format. In your dataset I'm seeing - `9876509.162535913,270606.1440290334` for `lat` - `lon` values. The correct formats are something like `36.939056099753,-1.1161267590278299`. Take a look at some other datasets in the repository for more information.
> 8. Please remove the ID column from your model-outputs.csv file
> 
> I think that's all for now - let me know if you'd like support with any of the above points.

andy-clarke-uofg commented:
Hi @wenlanzhang

Great to see this progress. We're getting there now. 

Some clarification on these points:

3. We don't need either the 'model-interpretability.html` file or the `models/waste-piles/nairobi/.ipynb_checkpoints/model-interpretability-checkpoint.md` file. Just one source of truth here is preferable. 

6. Yes absolutely - please see this `licensing-example.md` file that I've recently added -> https://github.com/urbanbigdatacentre/ideamaps-models/blob/dev/docs/datasets/licensing-example.md

I also notice one other point:

9. Your image file names do not follow the conventions set out here. https://github.com/urbanbigdatacentre/ideamaps-models/tree/dev/docs/datasets. Because you are using Low - Medium - High categories - we need the 3 square images to follow this naming convention: EG. `waste-piles-low.png` - `waste-piles-medium.png` - `waste-piles-high.png`
The other images you are using inside the `model-interpretability.md` file don't have to follow any particular convention.

Once this is done I am happy to merge the PR and close this work.

Thanks again,
Andy

wenlanzhang commented:
> Hi @wenlanzhang
> 
> Great to see this progress. We're getting there now.
> 
> Some clarification on these points:
> 
> 3. We don't need either the 'model-interpretability.html`file or the`models/waste-piles/nairobi/.ipynb_checkpoints/model-interpretability-checkpoint.md` file. Just one source of truth here is preferable.
> 4. Yes absolutely - please see this `licensing-example.md` file that I've recently added -> https://github.com/urbanbigdatacentre/ideamaps-models/blob/dev/docs/datasets/licensing-example.md
> 
> I also notice one other point:
> 
> 9. Your image file names do not follow the conventions set out here. https://github.com/urbanbigdatacentre/ideamaps-models/tree/dev/docs/datasets. Because you are using Low - Medium - High categories - we need the 3 square images to follow this naming convention: EG. `waste-piles-low.png` - `waste-piles-medium.png` - `waste-piles-high.png`
>    The other images you are using inside the `model-interpretability.md` file don't have to follow any particular convention.
> 
> Once this is done I am happy to merge the PR and close this work.
> 
> Thanks again, Andy

Hi Andy, 

I should have addressed all requested change. In terms of the licensing, I'm still not really sure, please feel free to change the file content if it doesn't make sense or let me know. 

Thank you so much for your feedbacks, and I'm so excited to see it available online! 

Best regards, 
Wenlan

andy-clarke-uofg commented:
Hi Wenlan - That's great I'm really happy with all the content so let me take it from here.

I will merge the PR in due course and get the ingestion started in the next couple of weeks. 

Have you seen my email from earlier - I'll paste the text here in case you haven't 


```
Hi Wenlan,
 
Hope you are doing well – I’m reaching out to ask if you would be happy to do a 10-minute review presentation of your Waste Piles model.
 
The main intention of this would be to show some of the model results in key areas and also to explain the interpretability document that goes alongside – specifically the three categories.
 
I’ve cc’ed in @Francis C. Onyambu who is leading the Nairobi specific meetings – we have an option to do this on Monday 7th at 1pm UK time.
 
Let me know if this is suitable.
 
Thanks,
Andy
```

Let me know if you are around for this and would be happy to present briefly. 

Many thanks again - also excited on my side :)

wenlanzhang commented:
> Hi Wenlan - That's great I'm really happy with all the content so let me take it from here.
> 
> I will merge the PR in due course and get the ingestion started in the next couple of weeks.
> 
> Have you seen my email from earlier - I'll paste the text here in case you haven't
> 
> ```
> Hi Wenlan,
>  
> Hope you are doing well – I’m reaching out to ask if you would be happy to do a 10-minute review presentation of your Waste Piles model.
>  
> The main intention of this would be to show some of the model results in key areas and also to explain the interpretability document that goes alongside – specifically the three categories.
>  
> I’ve cc’ed in @Francis C. Onyambu who is leading the Nairobi specific meetings – we have an option to do this on Monday 7th at 1pm UK time.
>  
> Let me know if this is suitable.
>  
> Thanks,
> Andy
> ```
> 
> Let me know if you are around for this and would be happy to present briefly.
> 
> Many thanks again - also excited on my side :)

Sure! more than happy to! Sorry for the delay. I was actually writing to reply you hahaha

wenlanzhang commented:
Hi Andy @andy-clarke-uofg , 
I just realised there's a mistake with the image, and I've fixed it. Please make sure use this updated version. Many thanks.
Best, 
Wenlan

andy-clarke-uofg reviewed (approved):
@wenlanzhang - thanks so much for all your hard work on this PR. I will now merge the resources with our main repository - dev branch! 🚢
